index.ts

```diff
@@ -0,0 +1,48 @@
+import countryList from './lib/country.json';
+import stateList from './lib/state.json';
+import cityList from './lib/city.json';
+
+var country_state_city = {
```

Comment:
export default

Reply:
I tried this earlier, I was getting some error.
Now it worked the same way.

May be changing require to import fixed it.
Am I right?

Reply:
I guess this comment https://github.com/harpreetkhalsagtbit/country-state-city/pull/12#discussion_r283086124 means you already found the solution for your question?